src/js/components/DateInput/doc.js

```diff
@@ -0,0 +1,60 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { getAvailableAtBadge } from '../../utils';
+
+export const doc = DateInput => {
+  const DocumentedDateInput = describe(DateInput)
+    .availableAt(getAvailableAtBadge('DateInput'))
+    .description('A control to input a single date or a date range.')
+    .usage(
+      `import { DateInput } from 'grommet';
+<DateInput id='item' name='item' />`,
```

Comment:
If `id` or `name` are not required props I'm not sure they should be part of this example.

Reply:
Just following the example in TextInput, TextArea, MaskedInput. It's an interesting question. Since not having a name causes unexpected issues when used with FormField, it seems more helpful to include it for first time users. I could go either way.

Reply:
For consistency let's use it as is. One more reminder for removing 'react-desc'...

Reply:
yep